l10n/configs/pontoon.toml

```diff
@@ -0,0 +1,121 @@
+# All files translated by the community by at least some locales should
+# be covered by the [[paths]] in this file.
+
+basepath = ".."
+
+locales = [
+    "ach",
+    "af",
+    "am",
+    "an",
+    "ar",
+    "ast",
+    "az",
+    "azz",
+    "be",
+    "bg",
+    "bn",
+    "br",
+    "bs",
+    "ca",
+    "cak",
+    "crh",
+    "cs",
+    "cy",
+    "da",
+    "de",
```

Comment:
Should `de` and `fr` be here if they are in `vendor.toml`?

Reply:
AFAICT, marketing doesn't intend to own *all* of mozilla.org, so yes.